vizro-ai/docs/pages/user-guides/genAI-use-case.md

```diff
@@ -0,0 +1,105 @@
+
+    !!! example "Vizro-AI chart"
+        === "Code for the cell"
+            ```py
+            import vizro_ai
+            from vizro_ai import VizroAI
+            import plotly.express as px
+            from dotenv import load_dotenv
+
+            load_dotenv()
+
+            df = px.data.gapminder()
+            vizro_ai = VizroAI(model_name="gpt-4-0613")
+
+            vizro_ai.plot(df,
+                          """Plot a bubble chart to show the changes in life expectancy
+                           and GDP per capita for each country over time.
+                           Color the bubbles by continent.
+                           Add animation on yearly basis, and do not use facets.
+                           Put the legend on top""")
+            ```
+        === "Result"
+            [![VizroAIChart]][VizroAIChart]
+
+        [VizroAIChart]: ../../assets/user_guides/vizro-ai-chart.gif
+
+2. Insert the resulting chart into a dashboard.
+
+    Once you are satisfied with the chart, you incorporate the chart into a [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard. In the explanation section from the result of the example above, you'll find the code snippet used to generate the chart.
+
+    ```py
+    @capture('graph')
+    def custom_chart(data_frame=pd.DataFrame()):
+        # Create a bubble chart
+        fig = px.scatter(data_frame, x='gdpPercap', y='lifeExp', animation_frame='year', animation_group='country',
+                         size='pop', color='continent', hover_name='country', log_x=True, size_max=60,
+                         range_y=[20,90],
+                         labels={'gdpPercap':'GDP per Capita', 'lifeExp':'Life Expectancy', 'year':'Year', 'continent':'Continent', 'pop':'Population'},
+                         title='Life Expectancy v. Per Capita GDP, 1952-2007',
+                         height=400)
+        # Put the legend on top
+        fig.update_layout(legend=dict(y=1, yanchor='top', x=0.5, xanchor='center'))
+        # Return the plot
+        return fig
+    ```
+
+The provided `custom_chart` function is an example of the [custom chart](https://vizro.readthedocs.io/en/stable/pages/user-guides/custom-charts), and it returns a `go.Figure()` object.
+This object must then be passed to the `figure` argument of the Vizro [Graph](https://vizro.readthedocs.io/en/stable/pages/user-guides/graph) model.
+
+!!! example "Vizro-AI chart within vizro dashboard"
+    === "Code for the cell"
+        ```py
+        from vizro import Vizro
+        import vizro.models as vm
+        from vizro.models.types import capture
+        import pandas as pd
+        import plotly.express as px
+
+
+        @capture('graph')
```

Comment:
I really like this approach. I put a similar very brief illustration in the core docs to show how it could be done if you just grab the call to Plotly and include that -- I'll add you to my PR to see what you think. Is it worth noting here that you could do that if you understand the code and want to minimize what you copy across?